### ISSUE TITLE
The conversion code to generate the BEV results

### ISSUE THREAD

**Issue by zeng32** [hlwang1124/SNE-RoadSeg#22]
Hi,

Could you please help me with the conversion from testresults to the BEV format before submitting it to the Kitti server?
I tried the devkit_road from the Kitti website, and I believe it needs to run under python 2.7.
Unfortunately, there is always some problem running it.
Would you please share your conversion code and execution steps with us if possible? It would be a great help!

Thanks in advance!

**Comment by hlwang1124** [comment]
Hi. I used the devkit_road from the KITTI website. I have also attached it [here](https://github.com/hlwang1124/SNE-RoadSeg/files/6097490/python.zip) for your reference. The used command is `python transform2BEV.py '/path_to_your_results/*.png' '/path_to_your_calibration_files' '/path_to_your_BEV_results'`.

**Comment by zeng32** [comment]
Thanks to your conversion code, I can almost replicate your reported accuracy using your shared pretrained model.
Unfortunately, after I train with the default parameters, I can only get:
MaxF:
95.12%, 97.07%, 94.69%, 95.88%
Would you please share the training parameter you adopt for training(with resnet152)?
Thanks again for your kind help!

**Comment by hlwang1124** [comment]
It can improve the performance a little to use some common data augmentation techniques. However, I think a more important technique for KITTI road submission is to select the used weights. I suggest you carefully split a validation set for weight selection.
